src/app/containers/ATIAnalytics/atiUrl/index.js

```diff
@@ -50,7 +50,9 @@ export const buildATIPageTrackPath = ({
   // For canonical, we have a requirement to encode the x5 and x6 value twice. Source issue: https://github.com/bbc/simorgh/pull/6593
   const x5Value = platform === 'amp' ? href : href && encodeURIComponent(href);
   const x6Value =
-    platform === 'amp' ? referrer : referrer && encodeURIComponent(referrer);
+    platform === 'amp'
```

Comment:
Does it make sense to create a new variable `const isAmp = platform === 'amp'` & use it here?

Reply:
I've done something similar in the follow-up PR, I'll put that out for review a bit later and see if I've addressed this point: https://github.com/bbc/simorgh/pull/7877